examples/configs/eiger.yaml

```diff
@@ -0,0 +1,3 @@
+- tickit.devices.eiger.Eiger:
+    inputs: {}
+    name: eiger
```

Comment:
We can't have a single device simulation, can we remove this until logic is in?

Reply:
This doesn't exist anymore, now it involves a Source too

Reply:
```yaml
- tickit.devices.source.Source:
    name: source
    inputs: {}
    value: 42.0
- tickit.devices.eiger.Eiger:
    inputs: 
      flux: source:value
    name: eiger```